carpool-map/js/geojson.js

```diff
@@ -0,0 +1,216 @@
+(function(GeoJSON) {
```

Comment:
This is a vendor file, right? Is there a minified version that we can use instead?